backend/graph/api/schema/pages.graphqls

```diff
@@ -12,15 +12,6 @@ type Page{
     ): SectionPagination! @goField(forceResolver: true)
 }
 
-type Collection {
```

Comment:
How does this impact the resulting API? I.e. Backwards compat for apps and for Africa?

Reply:
Did they use Collections?

I've never referenced this in the web code. Only place I know it was used is the old API for the search pages.

Reply:
The flutter apps doesnt use it

Reply:
Ok, then it should be fine.